### PR TITLE
Repl parse database name by the corresponding go driver

### PR DESCRIPTION
**Pull request by Yancey0623**

### CODE REVIEW

File: cmd/repl/repl.go [hunk 1]
```diff
@@ -249,6 +252,7 @@ func runStmt(stmt string, isTerminal bool, modelDir string, ds string) error {
 	table := tablewriter.NewWriter(os.Stdout)
 	sess := makeSessionFromEnv()
 	sess.DbConnStr = getDataSource(ds, currentDB)
+	fmt.Println("session.DBConnStr:", sess.DbConnStr)
```
Comment (shendiaomo) [inline cmd/repl/repl.go]: Does this message is only for debugging?

File: cmd/repl/repl.go [hunk 2]
```diff
@@ -396,9 +412,9 @@ func main() {
 	flag.StringVar(sqlFileName, "f", "", "execute SQLFlow from file, short for --file")
 	noAutoCompletion := flag.Bool("A", false, "No auto completion for sqlflow models. This gives a quicker start.")
 	flag.Parse()
-
 	assertConnectable(*ds) // Fast fail if we can't connect to the datasource
 	currentDB = getDatabaseName(*ds)
+	fmt.Println("currentDB: ", currentDB)
```
Comment (shendiaomo) [inline cmd/repl/repl.go]: Ditto.

File: cmd/repl/repl.go [hunk 3]
```diff
@@ -374,7 +389,7 @@ func getDataSource(dataSource, db string) string {
 			}
 		}
 		v["curr_project"] = []string{db}
-		return fmt.Sprintf("maxcompute://%s?%s", pieces[0], v.Encode())
+		return fmt.Sprintf("%s://%s?%s", driver, pieces[0], v.Encode())
```
Comment (weiguoz) [inline cmd/repl/repl.go:392]: Good catch!
Shall we add a UT here?

**Review by weiguoz — APPROVED** [review]
LGTM.
Thanks

**Pull request merged**